app/views/widgets/shared/_omninav.html

```diff
@@ -145,13 +145,13 @@
         <div class="cu-off-canvas-header">
             
               
-            <a href="/" class="sc-logo" title="Link to Chapman University Homepage">
+            <a href="/" class="sc-logo asbe" title="Link to Chapman University Homepage">
                 <span itemprop="name">Chapman University</span>
             </a>
                             
-            <a href="/" class="default-logo-cu" title="Link to Chapman University Homepage">
+            <!-- <a href="/" class="default-logo-cu" title="Link to Chapman University Homepage">
```

Comment:
Dead code should be removed. If there is a reason to keep it, add comment explaining why.